orc8r/cloud/go/swagger/swagger_spec.go

```diff
@@ -0,0 +1,38 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger
+
+// SwaggerSpec is the Go struct version of a OAI/Swagger 2.0 YAML spec file.
+type SwaggerSpec struct {
```

Comment:
Now that this struct is in pkg named `swagger`, we don't want to stutter -- that is, `swagger.Spec` is better than `swagger.SwaggerSpec`